vivisect/docs/switchcases.txt

```diff
@@ -0,0 +1,143 @@
+linux/i386/chgrp.llvm - no switchcases i can recognize.  all dynamic branches are calls.
```

Comment:
i'll clean up this notes file before merging.  right now it's tracking notes and details as i iterate through various switchcases already in our unittests.

Reply:
woops, and need to move it to vivisect/notes <facepalm>

Reply:
Ah, yes.